serverless-plugin-canary-deployments.js

```diff
@@ -207,6 +209,21 @@ class ServerlessCanaryDeployments {
     return getMappingsForFunction(this.compiledTpl.Resources);
   }
 
+  getSnsSubscriptionsFor(functionName) {
+    const isEventSourceMapping = (value) => {
```

Comment:
`_.matchesProperty` already returns a function, why are you wrapping it into an arrow function if you're not doing anything custom? Wouldn't something like this work?

```javascript
const isEventSourceMapping = _.matchesProperty('Type', 'AWS::SNS::Subscription');
const isSubscriptionForFunction = _.matchesProperty('Properties.Endpoint.Fn::GetAtt[0]', functionName);
```

Reply:
Use it to debug, i have changed to point to it directly instead